.eslintignore

```diff
@@ -1,4 +1,2 @@
 node_modules
-public/
-!.eslintrc.js
-!.stylelintrc.js
```

Comment:
as I know eslint ignores all files except eslint config. you shouldn't remove this line.

Reply:
all dot files*

Reply:
I don't think we need check `.stylelintrc.js`. It's just a config file like `.eslintrc.js`

Reply:
Why not check the configs?

Reply:
Because this configs just export plain JS objects. I don't see any profit or need in linting them.

Reply:
My point is: it is easy to implement, can show errors like shadowing in rules because of rule duplication, and simply keeps configs cleaner.

Reply:
Oh, I forgot that JS don't give a damn about keys duplication in objects. Okay then